### PR TITLE
Reverting OApackage to 2.6.8, so that miniconda3:4.9.2 can be used again

### LINKED ISSUE #265: Revert Docker image from `latest` to a tagged `build`
Currently, we build with the `continuumio/miniconda3:latest` to get the latest SWIG version, but hopefully this will become obsolete before the next release.

### PR DESCRIPTION
**Pull request by RandomDefaultUser**
I was wrong about OAPackage 2.6.8 not working. It does. So we can fix #265  by downgrading oapackage safely.

### CODE REVIEW

File: .github/workflows/cpu-tests.yml [hunk 1]
```diff
@@ -154,8 +154,12 @@ jobs:
           # epxort Docker image Conda environment for a later comparison
           conda env export -n mala-cpu > env_1.yml
 
+          # Manually install oapackage because we are bound to a certain
+          # version currently
+          pip install oapackage==2.6.8
+
           # install mala package
-          pip --no-cache-dir install -e .[opt,test]
+          pip --no-cache-dir install -e .[test]
```
[the remark this answers is not in the document]
Comment (DanielKotik) [reply .github/workflows/cpu-tests.yml:162]: Please also rebase your branch onto `develop` or merge `develop` into yours and ensure the pipeline is running successful.

[the remark this answers is not in the document]
Comment (RandomDefaultUser) [reply .github/workflows/cpu-tests.yml:162]: I can definitely do that, but I thought it be more consistent this way? As in, the reason we restrict ourselves to 2.6.8 is not  `mala` itself, as putting it in `setup.py` might suggest, but rather the CI, and so I put it in the CI. That being set, I can of course revert that.

[the remark this answers is not in the document]
Comment (DanielKotik) [reply .github/workflows/cpu-tests.yml:162]: Yeah you're right (it's already late...). I thought you changed something in the docs... :facepalm:, but rather you've made some changes in the CI. But eihter way, your changes are not need (please revert them back and also _do not_ apply my suggestion). We provide the correct version (2.6.8) within the docker image - in the CI, while executing `pip --no-cache-dir install -e .[opt,test]` founds this installation and will use it.

[the remark this answers is not in the document]
Comment (RandomDefaultUser) [reply .github/workflows/cpu-tests.yml:162]: Ah I see what you mean. You're right, that's really superfluous. I will revert the change, and also merge develop into this branch, so we can verify that everything is working correctly.

**Review by DanielKotik — APPROVED** [review]
LGTM now:)

**Pull request merged**